libraries/AP_ExternalAHRS/AP_ExternalAHRS.cpp

```diff
@@ -240,6 +241,25 @@ bool AP_ExternalAHRS::pre_arm_check(char *failure_msg, uint8_t failure_msg_len)
         hal.util->snprintf(failure_msg, failure_msg_len, "ExternalAHRS: No origin");
         return false;
     }
+
+    // Verify the user has configured the GPS to accept EAHRS data.
+    if (has_sensor(AvailableSensor::GPS)) {
+        const auto eahrs_gps_sensors = backend->num_gps_sensors();
+
+        uint8_t n_configured_eahrs_gps = 0;
+        for (uint8_t i = 0; i < GPS_MAX_INSTANCES; ++i) {
+            const auto gps_type = AP::gps().get_type(i);
```

Comment:
please take a reference, not in the loop:
```
  const auto &gps = AP::gps();
```

Reply:
Done, good tip.